aff4/aff4_imager_utils.cc

```diff
@@ -248,8 +248,7 @@ AFF4Status BasicImager::handle_aff4_volumes() {
                                     AFF4Flusher<AFF4Stream>(backing_stream.release()),
                                     volume));
 
-                volume_objs.AddVolume(
-                    std::move(AFF4Flusher<AFF4Volume>(volume.release())));
+                volume_objs.AddVolume(std::move(AFF4Flusher<AFF4Volume>(volume.release())));
```

Comment:
Remove the std::move from this call, it's not needed and will stop automatic copy elision